src/stores/modelRepositories/repository.js

```diff
@@ -129,6 +135,16 @@ export default class Repository {
 
       if(['acc', 'accp', 'meaniou', 'map', 'f1'].includes(metricKey)) {
         value = Math.max.apply(Math, measure_hist[metricHistKey])
+      } else if(['iteration'].includes(metricKey)) {
+        const bestTrainLossIndex = measure_hist["train_loss_hist"].indexOf(
```

Comment:
Why are you taking the best loss? The idea here would be to take the value stored in the best_model.txt file

Reply:
add `bestModel['iteration']` data fetching if exists in [a9d7e8c](https://github.com/jolibrain/platform_ui/pull/67/commits/a9d7e8cabc96f0aa6e189221f3864536bead7505)